setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup, find_packages
+
+setup(
```

Comment:
I think you might need an [`install_requires`](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies) in here. Some folks [use the `requirements.txt`](https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py) that they included